android/src/main/java/co/nimblehq/ic/kmm/suv/android/di/LoginModule.kt

```diff
@@ -1,9 +1,11 @@
 package co.nimblehq.ic.kmm.suv.android.di
 
+import co.nimblehq.ic.kmm.suv.android.ui.screens.home.HomeViewModel
 import co.nimblehq.ic.kmm.suv.android.ui.screens.login.LoginViewModel
 import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.dsl.module
 
 val loginModule = module {
     viewModel { LoginViewModel(get()) }
+    viewModel { HomeViewModel(get(), get(), get()) }
```

Comment:
I created `mainModule` for home-related stuff. Not sure about the naming, but with the mainModule, it will contain all view models for Home (Survey Lists) and Survey Detail. Normally, can we name `surveysModule`?

Fixed in https://github.com/suho/kmm-ic/pull/100/commits/5ad895b5ebe7f785e4dbbc060325c0509b5b89a7